tests/compat.sh

```diff
@@ -255,8 +255,6 @@ add_common_ciphersuites()
 
         "ECDSA")
             CIPHERS="$CIPHERS                           \
-                TLS-ECDHE-ECDSA-WITH-NULL-SHA           \
```

Comment:
I don't think `NULL` ciphersuites were removed in 3.0. Are you sure you used `config.py full` before building and running `compat.sh`?

Reply:
Ah maybe not, I will re-add them and double check

Reply:
Note: if `config.py full` causes issues for you as it does for me, `config.py set MBEDTLS_NULL_CIPHER` is enough to enable the `NULL` ciphersuites.

Reply:
with the null ciphers, both `config.py` and `config.py set MBEDTLS_NULL_CIPHER` have 72 tests skipped when running `compat.sh -e '^$' -p mbedtls`

Reply:
Sorry, I meant `MBEDTLS_CIPHER_NULL_CIPHER` - `config.py set MBEDTLS_NULL_CIPHER` probably told you that `MBEDTLS_NULL_CIPHER` was not the name of an actual config option.

Did you run `make clean` after changing the config? Our `Makefile`s don't have proper dependency handling, that could be the gotcha.

Anyway, from a clean checkout of current development, I ran:

Reply:
ah yeah that works, I will start ordering them now